browser/resources/settings/brave_default_extensions_page/brave_default_extensions_page.js

```diff
@@ -42,15 +43,18 @@ Polymer({
     this.onIPFSCompanionEnabledChange_ = this.onIPFSCompanionEnabledChange_.bind(this)
     this.openExtensionsPage_ = this.openExtensionsPage_.bind(this)
     this.openKeyboardShortcutsPage_ = this.openKeyboardShortcutsPage_.bind(this)
+    this.onWidevineEnabledChange_ = this.onWidevineEnabledChange_.bind(this)
     this.restartBrowser_ = this.restartBrowser_.bind(this)
-    this.onTorEnabledChange_ = this.onTorEnabledChange_.bind(this)
```

Comment:
was this removed intentionally?

Reply:
Oops. Thanks for catching @bsclifton ! added again.